### DISCUSSION TITLE
how can i install the app without it automatically changing settings?

### DISCUSSION THREAD

**Discussion by pwnlxke** [valinet/ExplorerPatcher#2978] [Q&A]
so when i install explorerpatcher and first open it, it will automatically change some settings instead of letting everything as it is and letting me change only what i want. i'd like to know how can i install it without touching what i dont want to

**Comment by pyrates999** [comment]
in the EP properties application, go to **About** and click on **Restore default settings**.  Then reboot.

**Reply by pwnlxke** [reply]
but this will only restore the default settings, and the default settings already change a lot of things
i want to install EP without changing anything on the computer until i choose to change it

**Reply by pyrates999** [reply]
When you uninstall EP, the settings you set are still there.  So restoring to default settings before you uninstall it is how you prevent this.
